apps/prairielearn/elements/pl-code/pl-code.mustache

```diff
@@ -10,9 +10,8 @@ $(function(){
     style="--pl-code-background-color: {{background_color}}; --pl-code-line-number-color: {{line_number_color}};">
     <div
         id="pl-code-{{uuid}}"
-        class="rounded"
+        class="rounded {{#prevent_select}}user-select-none{{/prevent_select}}"
         {{#prevent_select}}
-        style="-moz-user-select: none; -webkit-user-select: none; -ms-user-select:none; user-select:none;-o-user-select:none;"
         unselectable="on"
```

Comment:
Previous comment aside: AFAICT this is only for IE <8 so we can remove this one specifically if you want.